crispy_forms_materialize/templates/materialize_css_forms/layout/checkboxselectmultiple.html

```diff
@@ -1,7 +1,9 @@
 {% load l10n %}
-<ul>{% for choice in field.field.choices %}<li>
-    <label for="id_{{ field.html_name}}_{{ forloop.counter }}"{% if flat_attrs %} {{ flat_attrs|safe }}{% endif %}>
-        <input type="checkbox"{% if choice.0 in field.value or choice.0|stringformat:"s" in field.value or choice.0|stringformat:"s" == field.value|stringformat:"s" %} checked="checked"{% endif %} name="{{ field.html_name }}" id="id_{{ field.html_name }}_{{ forloop.counter }}" value="{{ choice.0|unlocalize }}">
+{% for choice in field.field.choices %}
+<p>
```

Comment:
@mmahkamov why you change an unordered list of elements by paragraphs?

Reply:
For some strange reason the latest Materialize expects the checkboxes to be enclosed within the p tag. Please see http://materializecss.com/forms.html for a sample layout in the "Checkboxes" section.
Otherwise the checkbox is rendered but not clickable.